src/vs/workbench/api/common/extHostApiCommands.ts

```diff
@@ -394,7 +394,7 @@ const newCommands: ApiCommand[] = [
 		'vscode.openWith', '_workbench.openWith', 'Opens the provided resource with a specific editor.',
 		[
 			ApiCommandArgument.Uri.with('resource', 'Resource to open'),
-			ApiCommandArgument.String.with('viewId', 'Custom editor view id or \'default\' to use VS Code\'s default editor'),
+			ApiCommandArgument.String.with('viewId', 'Custom editor view id. This should match the viewType from the customEditors contribution point. Use \'default\' to use VS Code\'s default editor'),
```

Comment:
This is custom editors but more than just that, there is also notebooks and likely others. Adding @lramos15 for the precise set of possible groups of view ids

Reply:
The main three I would document here is similar to the tabs api documentation. It's notebook's `notebookType` , custom editor's `viewId`, and for the text editor it's `default`

Reply:
@lramos15 The confusing part is that custom editors don't have something that's publicly called `viewId`. In all the other API documentation for creating a custom editor, it's called `viewType` and not `viewId`, right?

Reply:
![image](https://user-images.githubusercontent.com/1588988/158859910-c0ad2ae6-de1d-45a9-9d4e-ba693b9d6710.png)

Reply:
@jruales Yes `viewType` not `viewId` sorry. We have a mixed bag of words for this everywhere which is a bit annoying but `viewType`, `notebookType`, and `default` are the main things that can be passed into `viewId`

Reply:
@jrieken  I've updated the PR based on the feedback